dom.bs

```diff
@@ -3040,6 +3040,34 @@ standards that want to define APIs shared between <a for=/>documents</a> and
  <a for=CharacterData>data</a> is the string and <a for=Node>node document</a> is
  <var>document</var>.
 
+ <li><p>Let <var>isScriptElement</var> be <var>parent</var> is a {{HTMLScriptElement}}.
+
+ <li><p><a for=set>For each</a> <var>value</var> in <var>nodes</var>:
+  <ol>
+   <li>If <var>value</var> is a <a for=/>node</a>, then:
+    <ol>
+     <li>If <var>isScriptElement</var> is false, skip to the next <var>value</var>.
+     <li>If <var>value</var> is not a {{Text}} <a for=/>node</a>, then <a for=iteration>continue</a>.
+     <li>Let <var>text</var> be the result of executing the [$Get Trusted Type compliant string$] algorithm, with the following arguments:
+        * {{TrustedScript}} as <span>expectedType</span>,
+        * <var>document</var>'s [=relevant global object=] as <span>global</span>,
+        * <var>value</var>'s <a for=CharacterData>data</a> as <span>input</span>,
+        * `HTMLScriptElement text` as <span>sink</span>,
+        * `script` as <span>sinkGroup</span>.
```

Comment:
This invocation style is not described by Infra. It needs adjusting. We also don't use `<span>` in DOM in this way.

Reply:
This should hopefully be sorted. Not entirely sure what the "correct" way is.